src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+    public String description;
+    public Boolean isDone;
+    public String taskType;
+
+    Task() {
+        this.description = "";
+        this.isDone = false;
+    }
```

Comment:
uneven spacing between the two constructors as there is an empty new line between other functions